whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapControllerTestIntegration.java

```diff
@@ -3685,6 +3685,136 @@ public void get_bottom_wrong_type_then_400(){
+                        "netname:        test" +
+                        "descr:          issue causing IP\n" +
+                        "country:        EU\n" +
+                        "tech-c:         TP1-TEST\n" +
+                        "admin-c:       TP1-TEST\n" +
+                        "status:         ASSIGNED PA\n" +
+                        "mnt-by:         OWNER-MNT\n" +
+                        "created:       2011-07-28T00:35:42Z\n" +
+                        "last-modified: 2019-02-28T10:14:46Z\n" +
+                        "source:         TEST");
+
+        databaseHelper.addObject(
+                "inetnum:       37.210.96.1 - 37.210.99.254\n" +
+                        "netname:        test" +
+                        "descr:          issue causing IP\n" +
+                        "country:        EU\n" +
+                        "tech-c:         TP1-TEST\n" +
+                        "admin-c:       TP1-TEST\n" +
+                        "status:         ASSIGNED PA\n" +
+                        "mnt-by:         OWNER-MNT\n" +
+                        "created:       2011-07-28T00:35:42Z\n" +
+                        "last-modified: 2019-02-28T10:14:46Z\n" +
+                        "source:         TEST");
+
+        databaseHelper.addObject(
+                "inetnum:       37.210.128.1 - 37.210.255.254\n" +
+                        "netname:        test" +
+                        "descr:          issue causing IP\n" +
+                        "country:        EU\n" +
+                        "tech-c:         TP1-TEST\n" +
+                        "admin-c:       TP1-TEST\n" +
+                        "status:         ASSIGNED PA\n" +
+                        "mnt-by:         OWNER-MNT\n" +
+                        "created:       2011-07-28T00:35:42Z\n" +
+                        "last-modified: 2019-02-28T10:14:46Z\n" +
+                        "source:         TEST");
+
+        databaseHelper.addObject(
+                "inetnum:       37.211.0.1 - 37.211.63.254\n" +
+                        "netname:        test" +
+                        "descr:          issue causing IP\n" +
+                        "country:        EU\n" +
+                        "tech-c:         TP1-TEST\n" +
+                        "admin-c:       TP1-TEST\n" +
+                        "status:         ASSIGNED PA\n" +
+                        "mnt-by:         OWNER-MNT\n" +
+                        "created:       2011-07-28T00:35:42Z\n" +
+                        "last-modified: 2019-02-28T10:14:46Z\n" +
+                        "source:         TEST");
+
+        ipTreeUpdater.rebuild();
+
+        final SearchResult searchResult = createResource("ips/rirSearch1/bottom/37.210.0.0%20-%2037.211.255.255")
+                    .request(MediaType.APPLICATION_JSON_TYPE)
+                    .get(SearchResult.class);
+
+        final List<Ip> ipResults = searchResult.getIpSearchResults();
+        assertThat(ipResults.size(), is(8));
```

Comment:
Should you test that the cross-prefix range is included? Not just the size of the response. Maybe also remove some of the unnecessary inetnums if they are not being tested.

Reply:
Yes, good idea